app/models/digital_object/base.rb

```diff
@@ -21,7 +21,7 @@ class DigitalObject::Base
   
   def require_subclass_override!; raise 'This method must be overridden by a subclass'; end
   
-  def initialize()
+  def initialize(*args)
```

Comment:
Because the call to super in the subclasses bombs if you pass the args hash.

On Mon, Dec 21, 2015 at 2:12 PM, Eric O <notifications@github.com> wrote:

> In app/models/digital_object/base.rb
> <https://github.com/cul/hyacinth/pull/8#discussion_r48179142>:
>
> > @@ -21,7 +21,7 @@ class DigitalObject::Base
> >
> >    def require_subclass_override!; raise 'This method must be overridden by a subclass'; end
> >
> > -  def initialize()
> > +  def initialize(*args)
>
> Why add (*args) if this method doesn't take any arguments?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cul/hyacinth/pull/8/files#r48179142>.
>